client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -5,17 +5,19 @@ import StarRating from '../common/starRating.jsx';
 import './styles.css';
 
 const ProductCard = (props) => {
-  const productCategory = props.product.category.toUpperCase();
+  const { product } = props.product;
+  const productCategory = product.category.toUpperCase();
+
   return (
     <div className='card'>
-      <Button type={ props.type } />
-      <img src='' alt={ props.product.name } />
+      <Button type={ props.type } product={ props.product } onClickStar={ props.onClickStar } />
```

Comment:
Would `product={product}` work since you defined it above?

Reply:
Good catch!  Thanks!